brian2/codegen/codeobject.py

```diff
@@ -209,7 +209,7 @@ def create_runner_codeobj(group, code, template_name,
         Whether to check units in the statement. Defaults to ``True``.
     needed_variables: list of str, optional
         A list of variables that are neither present in the abstract code, nor
-        in the ``USES_VARIABLES`` statement in the template. This is only
+        in the ``USES_VARIABLES`` statement in theqq template. This is only
```

Comment:
Hah indeed, was working with two computers and two keyboards, pressing a key on one that was meant for the other one happened more than once :)